tests/foreman/cli/test_variables.py

```diff
@@ -288,12 +422,26 @@ def test_positive_create_empty_matcher_value(self):
         @steps: Create a matcher for variable with type string and empty value
 
         @assert: Matcher is created with empty value
-
-        @caseautomation: notautomated
         """
+        smart_variable = make_variable({
+            'variable': gen_string('alpha'),
+            'puppet-class': self.puppet['name'],
+            'default-value': gen_string('alpha'),
+        })
+        SmartVariable.add_override_value({
+            'smart-variable-id': smart_variable['id'],
+            'match': 'is_virtual=true',
+            'value': '',
+        })
+        smart_variable = SmartVariable.info({'id': smart_variable['id']})
+        self.assertEqual(
+            smart_variable['override-values']['values']['1']['match'],
```

Comment:
Ugh, that smells bad. Is that an issue with our parser or the info is returned in that weird way?